modules/timepoint_list/test/TestPlan.md

```diff
@@ -2,7 +2,10 @@
 
 1.  **Module access permissions**
     - For a candidate of the same site as your user, accessing the timepoint_list module should not require any permission.
-    - For a candidate of a different site than your user, ensure that `access_all_profiles` permission is required.
+    - For a candidate of a different site than your user, ensure that either 
+        - `access_all_profiles` permission is required 
+        - or that at least one of the candidate's sessions was performed at the 
```

Comment:
I dont think that thats what the current code is doing. it seems like it's only if the candidate's registration site is in the list of sites of the user

Reply:
Right. Modifying the documentation accordingly.